.github/workflows/build-wheel.yml

```diff
@@ -102,53 +116,23 @@ jobs:
           python -c "import mlflow; print(mlflow.__version__)"
 
       # Anyone with read access can download the uploaded wheel on GitHub.
-      - name: Store wheel
+      - name: Upload wheel
         uses: actions/upload-artifact@v4
-        if: github.event_name == 'push'
+        # if: github.event_name == 'workflow_dispatch'
+        id: upload-wheel
         with:
           name: ${{ steps.build-dist.outputs.wheel-name }}
           path: ${{ steps.build-dist.outputs.wheel-path }}
+          retention-days: 7
+          if-no-files-found: error
```

Comment:
Should we just finish rather than errors out when it is on PR run? Uploading wheel can fail due to github issue or quota breach, which should not block PR.

Reply:
Once https://github.com/mlflow/mlflow/pull/13356/files#r1793402256 is applied, this workflow only uploads when manually triggered :)

Reply:
Even if we hit the artifact storage, GitHub automatically removes old artifacts.